lib/Secrets/doc/sailfish-secrets-overview.qdoc

```diff
@@ -203,15 +199,14 @@ a collection is very similar to that for deleting a secret, so only the
 latter will be shown below.
 
 \code
-QDBusPendingReply<Sailfish::Secrets::Result>
-        deleteReply = sm.deleteSecret(
-                Sailfish::Secrets::Secret::Identifier("ExampleSecret",
-                                                      "ExampleCollection"),
-                SecretManager::ApplicationInteraction);
-deleteReply.waitForFinished();
-if (deleteReply.argumentAt<0>().code() == Sailfish::Secrets::Result::Failed) {
+Sailfish::Secrets::DeleteCollectionRequest dcr(&sm);
+dcr.setCollectionName(QLatin1String("ExampleCollection"));
+dcr.setUserInteractionMode(Sailfish::Secrets::SecretManager::SystemInteraction);
+dcr.startRequest();
+dcr.waitForFinished();
```

Comment:
I have already told about `performRequest`, I see there is a method for waiting, but it is still better from user-point of view to have a separate method, for example, `performRequest()` which simply calls `startRequest()` and then `waitForFinished()`.